pkg/controllers/disruption/consolidation.go

```diff
@@ -192,6 +193,52 @@ func (c *consolidation) computeConsolidation(ctx context.Context, candidates ...
 		results.NewNodeClaims[0].Requirements.Add(scheduling.NewRequirement(v1beta1.CapacityTypeLabelKey, v1.NodeSelectorOpIn, v1beta1.CapacityTypeSpot))
 	}
 
+	results.NewNodeClaims[0].NodeClaimTemplate.InstanceTypeOptions = instanceTypesOptionsFilteredByPrice
+	return Command{
+		candidates:   candidates,
+		replacements: results.NewNodeClaims,
+	}, nil
+}
+
+// Compute command to execute spot-to-spot consolidation if:
+// 1. The SpotToSpotConsolidation feature flag is set to true.
+// 2. There are atleast 15 cheapest instance type replacement options to consolidate.
```

Comment:
```suggestion
// 2. There are at least 15 cheapest instance type replacement options to consolidate.
```

nit

Reply:
Addressed.

Reply:
Looks like your commit might not have gone through on this one @nikmohan123

Reply:
Yeah, I am yet to push this file and it has this change. I accidentally published the comments for this one before pushing the changes. We can keep this one open till my commit reflects this. Thanks.

Reply:
Addressed.